app/src/main/java/io/bankingofthings/iot/Finn.kt

```diff
@@ -262,8 +269,12 @@ class Finn(
      * Stops bluetooth
      */
     fun stop() {
```

Comment:
I think this comment should be removed, cause method is doing more things. Or full description should be added.